src/components/nav.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+ <div class="nav_wrapper">
+
+    <div class="nav_changer">
+     <div><p>Home</p></div>
+     <div  class="active"><p>Workspace</p></div>
+     <div><p>Others</p></div>
+    </div>
+
+    <div class="nav_iconbox">
+      <img src="http://serwer1982875.home.pl/img-bright/search_iconn.svg" alt="search icon">
```

Comment:
Are you sure that using temporary server is a good idea? Better will be use assets or some ready components with sag icon

Reply:
I use the best option for me.  When we have serve  change this.

Reply:
You can download these files (if license allows you of course) and store it in assets directory